Refactor DiagnosticSource docs. (#4140)
getsentry/sentry-docs: src/includes/performance/automatic-instrumentation-integrations/dotnet.mdx

## DiagnosticSource Integration

> Starting with version 3.9.0, the SDK will automatically integrate with Entity Framework Core and SQLClient whenever available. Those integrations will automatically be activated if your project matches one of the following conditions:

Reviewer: ```suggestion
Starting with version 3.9.0, the SDK automatically integrates with Entity Framework Core and SQLClient whenever available. Those integrations are automatically activated if your project matches one of the following conditions:
```

* Includes `Sentry.AspNet` 3.9.0 or higher
* Includes `Sentry.AspNetCore` 3.9.0 or higher

> * Targets .NET Core 3.0 or higher (for example .NET 5)

Reviewer: ```suggestion
* Targets .NET Core 3.0 or higher (for example, .NET 5)
```

> Even if your project doesn't meet any of those requirements, you'll be able to manually activate those instrumentations by including the package 'Sentry.DiagnosticSource' and enabling it during on the SDK's initialization.

Reviewer: Do you mean for Sentry.DiagnosticSource to be in code style or in quotation marks? If you want quotation marks, use double quotes rather than single ones. I've edited it to be in code style, because I wasn't sure which you meant.

```suggestion
Even if your project doesn't meet any of those requirements, you can manually activate those instrumentations by including the package `Sentry.DiagnosticSource` and enabling it during on the SDK's initialization.
```

> Starting with version 3.9.0, the SDK automatically integrates with Entity Framework Core and SQLClient whenever available. Those integrations are automatically activated if your project matches one of the following conditions:
> * Includes `Sentry.AspNet` 3.9.0 or higher
> * Includes `Sentry.AspNetCore` 3.9.0 or higher
> * Targets .NET Core 3.0 or higher (for example, .NET 5)
> 
> Even if your project doesn't meet any of those requirements, you can manually activate those instrumentations by including the package `Sentry.DiagnosticSource` and enabling it during on the SDK's initialization.
> 
> ```csharp
> // Requires NuGet package: Sentry.DiagnosticSource
> using Sentry;
> 
> option.AddDiagnosticSourceIntegration();
> 
> ```
> 
> If you don't want to have this integration, you can disable it on `SentryOptions` by calling `DisableDiagnosticSourceIntegration();`
> ```csharp
> using Sentry;
> 
> option.DisableDiagnosticSourceIntegration();
> ```
> 

Reviewer 2: ````suggestion
Starting with version 3.9.0, the SDK automatically integrates with Entity Framework Core and SQLClient whenever available. Those integrations are automatically activated if your project matches one of the following conditions:

* Includes `Sentry.AspNet` 3.9.0 or higher
* Includes `Sentry.AspNetCore` 3.9.0 or higher
* Includes `Sentry` 3.9.0 and targets .NET Core 3.0 or higher (for example, .NET 5)

If you don't want to have this integration, you can disable it on `SentryOptions` by calling `DisableDiagnosticSourceIntegration();`

```csharp
using Sentry;

option.DisableDiagnosticSourceIntegration();
```

If your project doesn't match any of the conditions above, (for example, it targets .NET Framework 4.6.1 and uses only the `Sentry` package), you can still manually activate those instrumentations by including the package `Sentry.DiagnosticSource` and enabling it during on the SDK's initialization.

```csharp
// Requires NuGet package: Sentry.DiagnosticSource
using Sentry;

option.AddDiagnosticSourceIntegration();
```
````
